### ISSUE TITLE
AUR compile error

### ISSUE THREAD

**Issue by LinuxPony** [myszha/qt-ponies#15]
On attempting to install using the PKGBUILD from the AUR i get this error
message:

```
/usr/include/qt4/phonon -I/usr/include/qt4
-I/usr/include/qt4/phonon_compat -Isrc/moc -Isrc/ui -o bin/speak.o
src/speak.cpp
src/speak.cpp:23:31: fatal error: Phonon/MediaObject: No such file or
directory
  #include <Phonon/MediaObject>
                               ^
compilation terminated.
Makefile:331: recipe for target 'bin/speak.o' failed
make: *** [bin/speak.o] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```

MediaObject is present under /usr/include/qt4/phonon which is what it
appears to be looking for, could this be a problem with the casing since
it's expecting Phonon/Mediaobject when the directory is
phonon/MediaObject?

Running ArchLinux with Linux 3.13.5-1-ARCH x86_64

Any help would be greatly appreciated.

**Comment by myszha** [comment]
Phonon support is not really working (only supports a single sound at a time). The problem here is a different layout of include files for it than in Debian. I've fixed it in ce8c09bbf7, and disabled Phonon support by default (you will have to uncomment a line in qt-ponies.pro for it to include it), as the sound files are not included right now. You can manually add the sound files, if you would like to experiment, from the Desktop Ponies package here: http://desktopponies.bugs3.com/

**Comment by LinuxPony** [comment]
Thank you for the reply, the install worked now :-)
